packages/core/core/src/utils/template_generator.js

```diff
@@ -260,7 +265,7 @@ class TemplateGenerator {
       let templateAndBranch = uri.split('#');
       if (templateAndBranch.length === 1) {
         fallback = true;
-        embarkVersion = semver(require('../../../package.json').version);
+        embarkVersion = semver(require('../../../../../package.json').version);
```

Comment:
It is possible for the core package's version to lag behind `packages/embark`, i.e. if in a minor/patch bump the core package didn't change but `packages/embark` did. So I suggest this instead:

```suggestion
        embarkVersion = semver(require(embarkPath('package.json')).version);
```

Reply:
Thanks, will do!